src/nn/webgl/WebGLModel.js

```diff
@@ -555,7 +548,8 @@ export default class WebGLModel {
         const targetShape = operands[inputs[1]];
         const output = operands[outputs[0]];
         if (targetShape.value === undefined) {
-          targetShape.value = targetShape.dataSync();
+          const operand = this._model._operands[inputs[1]];
+          targetShape.value = Array.apply([], operand.value);
```

Comment:
Can you remind me what's the type of `operand`? Why do we need to set `targetShape.value = Array.apply([], operand.value);`?